index.js

```diff
@@ -133,6 +135,17 @@ class protractorImageComparison {
     this.toolBarShadowPadding = 6;
     this.viewPortHeight = 0;
 
+    if(options.clearFolder) {
+        rimraf(this.actualFolder , (e) => {
```

Comment:
You are now adding a new dependency, but we can use the existing libraries to remove the folder with `removeSync` of `fs-extra`, see [here](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove-sync.md)

Can you change this?

Reply:
thanks for letting me know. i just pushed the changes. please have a look.